h/security/predicates.py

```diff
@@ -10,10 +10,15 @@
 group.
 """
 
+from __future__ import annotations
```

Comment:
This avoids having to put `"EditGroupMembershipContext"` and `"GroupMembershipContext"` in quotes in type annotations when using `if TYPE_CHECKING`.